css/src/tokens/themes.scss

```diff
@@ -82,6 +82,7 @@ $themes: (
 		'info-active': $palette-purple-70,
 		'info-box-shadow': $palette-purple-opacity-30,
 		'info-invert': $palette-white,
+		'info-vivid': $palette-purple-c,
```

Comment:
`primary`, `info`, `warning` etc should all share the same structure and the same names. I think that maybe we should add a new set of properties to each of the themes that correspond to the start and end of the new gradient. Something like the following: 

`gradient-argon-start: $palette-value`
`gradient-argon-midde: $palette-value`
`gradient-argon-end: $palette-value`

Side note: not sure about `argon`, since it implies (I think) blue, but these should be conceptual not color specific names.